pkg/policy/controller.go

```diff
@@ -305,7 +305,7 @@ func (pc *PolicyController) syncPolicy(key string) error {
 	}
 
 	// if the policy contains mutating & validation rules and it config does not exist we create one
-	if policy.HasMutateOrValidate() {
+	if policy.HasMutateOrValidateOrGenerate() {
```

Comment:
By default, a valid policy must have one of mutate, validate, generate rule. Any possibility that a policy could exist without any of the rules defined?

Reply:
Not as of now, removing it